### PR TITLE
add 8.4 in basic features

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

Add 8.4 in basic features
  - Newly added experimental features in v8.4.0:
    - Vectort types
    - Vector functions and operators
    - Vector index
    - Vector search
    - Instance-level execution plan cache
  - Changes from experimental to GA in v8.4.0:
    - Global index
  - Removed in v8.4.0:
    - TiDB Binlog

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md [hunk 1]
```diff
@@ -20,247 +20,247 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [前台限流](/tikv-configuration-file.md#前台限流) | Y | Y | Y | Y | Y | Y | Y | E | N | N | N | N |
+| [后台限流](/tikv-configuration-file.md#后台限流) | E | E | E | E | E | E | E | N | N | N | N | N |
+| [基于 EBS 的备份和恢复](https://docs.pingcap.com/zh/tidb-in-kubernetes/v1.4/volume-snapshot-backup-restore) | Y | Y | Y | Y | Y | Y | Y | N | N | N | N | N |
+| [PITR](/br/br-pitr-guide.md) | Y | Y | Y | Y | Y | Y | Y | N | N | N | N | N |
+| [全局内存控制](/configure-memory-usage.md#如何配置-tidb-server-实例使用内存的阈值) | Y | Y | Y | Y | Y | Y | Y | N | N | N | N | N |
+| [RawKV 跨集群复制](/tikv-configuration-file.md#api-version-从-v610-版本开始引入) | E | E | E | E | E| E | E | N | N | N | N | N |
+| [Green GC](/system-variables.md#tidb_gc_scan_lock_mode-从-v50-版本开始引入) | E | E | E | E | E | E | E | E | E | E | E | E |
+| [资源管控 (Resource Control)](/tidb-resource-control.md) | Y | Y | Y | Y | Y | Y | N | N | N | N | N | N |
+| [Runaway Queries 自动管理](/tidb-resource-control.md#管理资源消耗超出预期的查询-runaway-queries) | Y | Y | Y | Y | E | N | N | N | N | N | N | N |
+| [后台任务资源管控](/tidb-resource-control.md#管理后台任务) | E | E | E | E | E | N | N | N | N | N | N | N |
```
Comment (hfxsd) [inline basic-features.md:255]:
```suggestion
| [后台任务资源管控](/tidb-resource-control.md#管理后台任务) | Y | E | E | E | E | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 2]
```diff
@@ -20,247 +20,247 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [不可见索引](/sql-statements/sql-statement-create-index.md#不可见索引) | Y | Y | Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [复合主键](/constraints.md#主键约束) | Y | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [`CHECK` 约束](/constraints.md#check-约束) | Y | Y |  Y |  Y |  Y  | N | N | N | N | N | N | N |
+| [唯一约束](/constraints.md#唯一约束) | Y | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [整型主键上的聚簇索引](/clustered-indexes.md) | Y | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [复合或非整型主键上的聚簇索引](/clustered-indexes.md) | Y | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [多值索引](/sql-statements/sql-statement-create-index.md#多值索引) | Y | Y |  Y |  Y |  Y  | Y | N | N | N | N | N | N |
+| [外键约束](/constraints.md#外键约束) | E | E | E |  E |  E  | E | N | N | N | N | N | N |
+| [TiFlash 延迟物化](/tiflash/tiflash-late-materialization.md) | Y | Y |  Y |  Y |  Y  | Y | N | N | N | N | N | N |
+| [全局索引 (Global Index)](/partitioned-table.md#全局索引) | E | E |  N |  N | N  | N | N | N | N | N | N | N |
```
Comment (hfxsd) [inline basic-features.md:65]:
```suggestion
| [全局索引 (Global Index)](/partitioned-table.md#全局索引) | Y | E |  N |  N | N  | N | N | N | N | N | N | N |
```

File: basic-features.md
Comment (Oreoxmt) [inline basic-features.md:1]: preview: https://pingcap-docsite-preview-git-v84-preview-oreoxmts-projects.vercel.app/zh/tidb/dev/basic-features

File: basic-features.md [hunk 3]
```diff
@@ -20,247 +20,247 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [MPP 执行引擎 - compression exchange](/explain-mpp.md#mpp-version-和-exchange-数据压缩) | Y |  Y  |  Y  |  Y  | Y | N | N | N | N | N | N |
-| [TiFlash Pipeline 执行模型](/tiflash/tiflash-pipeline-model.md) | Y |  Y  |  Y  |  Y  | N | N | N | N | N | N | N |
-| [TiFlash 副本选择策略](/system-variables.md#tiflash_replica_read-从-v730-版本开始引入) | Y |  Y  |  Y  |  Y  | N | N | N | N | N | N | N |
-| [索引合并](/explain-index-merge.md) | Y |  Y  |  Y  |  Y  | Y | Y | Y | Y | E | E | E |
-| [基于 SQL 的数据放置规则](/placement-rules-in-sql.md) | Y |  Y  |  Y  |  Y  | Y | Y | Y | E | E | N | N |
-| [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner) |  E  |  E  |  E  |  E  | E | E | E | E | E | E | E |
-| [Runtime Filter](/runtime-filter.md) | Y |  Y  |  Y  |  Y  | N | N | N | N | N | N | N |
+| 高级 SQL 功能 | 8.4 | 8.3 | 8.2 | 8.1 | 7.5 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [Prepare 语句执行计划缓存](/sql-prepared-plan-cache.md) | Y | Y | Y |  Y  |  Y  | Y | Y | Y | Y | Y | E | E |
```
Comment (songrijie) [inline basic-features.md:96]:
```suggestion
| [Prepare 语句执行计划缓存](/sql-prepared-plan-cache.md) | Y | Y | Y |  Y  |  Y  | Y | Y | Y | Y | Y | E | E |
| 实例级执行计划缓存 | E | N | N |  N  |  N  | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 4]
```diff
@@ -20,247 +20,247 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [DM](/dm/dm-overview.md) 支持迁移 MySQL 8.0  | Y | Y | Y | E | E | E | E | N | N | N | N |
+| 数据导入和导出 | 8.4 | 8.3 | 8.2 | 8.1 | 7.5 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [快速导入 (TiDB Lightning)](/tidb-lightning/tidb-lightning-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [快速导入 (`IMPORT INTO`)](/sql-statements/sql-statement-import-into.md) | Y | Y | Y | Y | Y | N | N | N | N | N | N | N |
+| mydumper 逻辑导出 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 |
+| [Dumpling 逻辑导出](/dumpling-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [事务 `LOAD DATA`](/sql-statements/sql-statement-load-data.md) [^5] | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [数据迁移工具](/migration-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [TiDB Binlog](https://docs.pingcap.com/zh/tidb/v8.3/tidb-binlog-overview) [^6] | 已废弃 | 已废弃 | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
```
Comment (lilin90) [inline basic-features.md:216]:
```suggestion
| [TiDB Binlog](https://docs.pingcap.com/zh/tidb/v8.3/tidb-binlog-overview) [^6] | 已移除 | 已废弃 | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
```

File: basic-features.md [hunk 5]
```diff
@@ -20,247 +20,247 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [其他函数](/functions-and-operators/miscellaneous-functions.md) |  Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
-| [操作符](/functions-and-operators/operators.md) |  Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
-| [字符集和排序规则](/character-set-and-collation.md) [^1] |  Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
-| [用户级别锁](/functions-and-operators/locking-functions.md) |  Y |  Y |  Y |  Y  | Y | Y | Y | N | N | N | N |
+| 数据类型，函数，操作符 | 8.4 | 8.3 | 8.2 | 8.1 | 7.5 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [数值类型](/data-type-numeric.md) |  Y |  Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [日期和时间类型](/data-type-date-and-time.md) |  Y |  Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [字符串类型](/data-type-string.md) |  Y |  Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [JSON 类型](/data-type-json.md) |  Y |  Y |  Y |  Y |  Y  | Y | Y | E | E | E | E | E |
```
Comment (qiancai) [inline basic-features.md:28]:
```suggestion
| [JSON 类型](/data-type-json.md) |  Y |  Y |  Y |  Y |  Y  | Y | Y | E | E | E | E | E |
| [向量数据类型](/vector-search-data-types.md) |  E |  N |  N |  N |  N  | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 6]
```diff
@@ -20,247 +20,247 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [控制流程函数](/functions-and-operators/control-flow-functions.md) |  Y |  Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [字符串函数](/functions-and-operators/string-functions.md) |  Y |  Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [数值函数与操作符](/functions-and-operators/numeric-functions-and-operators.md) |  Y |  Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [日期和时间函数](/functions-and-operators/date-and-time-functions.md)|  Y  |  Y  |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [位函数和操作符](/functions-and-operators/bit-functions-and-operators.md) |  Y |  Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [Cast 函数和操作符](/functions-and-operators/cast-functions-and-operators.md) |  Y |  Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [加密和压缩函数](/functions-and-operators/encryption-and-compression-functions.md) |  Y |  Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
```
Comment (qiancai) [inline basic-features.md:35]:
```suggestion
| [加密和压缩函数](/functions-and-operators/encryption-and-compression-functions.md) |  Y |  Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
| [向量函数和操作符](/vector-search-functions-and-operators.md) |  E |  N |  N |  N |  N  | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 7]
```diff
@@ -20,247 +20,247 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [MPP 执行引擎](/explain-mpp.md) | Y |  Y  |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y |
-| [MPP 执行引擎 - compression exchange](/explain-mpp.md#mpp-version-和-exchange-数据压缩) | Y |  Y  |  Y  |  Y  | Y | N | N | N | N | N | N |
-| [TiFlash Pipeline 执行模型](/tiflash/tiflash-pipeline-model.md) | Y |  Y  |  Y  |  Y  | N | N | N | N | N | N | N |
-| [TiFlash 副本选择策略](/system-variables.md#tiflash_replica_read-从-v730-版本开始引入) | Y |  Y  |  Y  |  Y  | N | N | N | N | N | N | N |
-| [索引合并](/explain-index-merge.md) | Y |  Y  |  Y  |  Y  | Y | Y | Y | Y | E | E | E |
-| [基于 SQL 的数据放置规则](/placement-rules-in-sql.md) | Y |  Y  |  Y  |  Y  | Y | Y | Y | E | E | N | N |
-| [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner) |  E  |  E  |  E  |  E  | E | E | E | E | E | E | E |
-| [Runtime Filter](/runtime-filter.md) | Y |  Y  |  Y  |  Y  | N | N | N | N | N | N | N |
+| 高级 SQL 功能 | 8.4 | 8.3 | 8.2 | 8.1 | 7.5 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
```
Comment (qiancai) [inline basic-features.md:95]:
```suggestion
|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
| [向量搜索](/vector-search-overview.md) |  E |  N |  N |  N |  N  | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 8]
```diff
@@ -20,247 +20,247 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [不可见索引](/sql-statements/sql-statement-create-index.md#不可见索引) | Y | Y | Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [复合主键](/constraints.md#主键约束) | Y | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [`CHECK` 约束](/constraints.md#check-约束) | Y | Y |  Y |  Y |  Y  | N | N | N | N | N | N | N |
+| [唯一约束](/constraints.md#唯一约束) | Y | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [整型主键上的聚簇索引](/clustered-indexes.md) | Y | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [复合或非整型主键上的聚簇索引](/clustered-indexes.md) | Y | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [多值索引](/sql-statements/sql-statement-create-index.md#多值索引) | Y | Y |  Y |  Y |  Y  | Y | N | N | N | N | N | N |
+| [外键约束](/constraints.md#外键约束) | E | E | E |  E |  E  | E | N | N | N | N | N | N |
+| [TiFlash 延迟物化](/tiflash/tiflash-late-materialization.md) | Y | Y |  Y |  Y |  Y  | Y | N | N | N | N | N | N |
+| [全局索引 (Global Index)](/partitioned-table.md#全局索引) | Y | E |  N |  N | N  | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:65]:
```suggestion
| [向量索引](/vector-search-index.md) |  E |  N |  N |  N |  N  | N | N | N | N | N | N | N |
```

Comment (qiancai) [inline basic-features.md:65]:
```suggestion
| [全局索引 (Global Index)](/partitioned-table.md#全局索引) | Y | E |  N |  N | N  | N | N | N | N | N | N | N |
| [向量索引](/vector-search-index.md) |  E |  N |  N |  N |  N  | N | N | N | N | N | N | N |
```

**Review by joey-yez — APPROVED** [review]

File: basic-features.md [hunk 9]
```diff
@@ -20,247 +20,252 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [索引合并](/explain-index-merge.md) | Y |  Y  |  Y  |  Y  | Y | Y | Y | Y | E | E | E |
-| [基于 SQL 的数据放置规则](/placement-rules-in-sql.md) | Y |  Y  |  Y  |  Y  | Y | Y | Y | E | E | N | N |
-| [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner) |  E  |  E  |  E  |  E  | E | E | E | E | E | E | E |
-| [Runtime Filter](/runtime-filter.md) | Y |  Y  |  Y  |  Y  | N | N | N | N | N | N | N |
+| 高级 SQL 功能 | 8.4 | 8.3 | 8.2 | 8.1 | 7.5 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [向量搜索](/vector-search-overview.md) |  E |  N |  N |  N |  N  | N | N | N | N | N | N | N |
+| [Prepare 语句执行计划缓存](/sql-prepared-plan-cache.md) | Y | Y | Y |  Y  |  Y  | Y | Y | Y | Y | Y | E | E |
+| [实例级执行计划缓存](/system-variables.md#tidb_enable_instance_plan_cache-从-v840-版本开始引入) | E | N | N |  N  |  N  | N | N | N | N | N | N | N |
+| [非 Prepare 语句执行计划缓存](/sql-non-prepared-plan-cache.md) | Y | Y |  Y  |  Y  |  Y  | E | N | N | N | N | N | N |
```
Comment (Oreoxmt) [inline basic-features.md:102]:
```suggestion
| [Prepare 语句执行计划缓存](/sql-prepared-plan-cache.md) | Y | Y | Y |  Y  |  Y  | Y | Y | Y | Y | Y | E | E |
| [非 Prepare 语句执行计划缓存](/sql-non-prepared-plan-cache.md) | Y | Y |  Y  |  Y  |  Y  | E | N | N | N | N | N | N |
| [实例级执行计划缓存](/system-variables.md#tidb_enable_instance_plan_cache-从-v840-版本开始引入) | E | N | N |  N  |  N  | N | N | N | N | N | N | N |
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**